nosso-calendario/src/main/java/com/br/devefc/nossocalendario/helpers/EncodeSenha.java

```diff
@@ -0,0 +1,21 @@
+package com.br.devefc.nossocalendario.helpers;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+
+public class EncodeSenha {

```

Comment:
*Deixamos pistas que facilitem o uso do código onde não conseguimos resolver com compilação.*

Ficou bem legal. O nome podia ser um pouco mais semântico, mas já facilitou bastante!

Reply:
Como eu posso deixar mais semântico ?

Reply:
Acabei de subir o vídeo lá no canal :) => https://youtu.be/AivMq1vhK_Q